python_modules/libraries/dagster-dg/dagster_dg/cli/init.py

```diff
@@ -43,7 +43,12 @@ def init_command(
         default=DEFAULT_WORKSPACE_NAME,
     ).strip()
 
-    workspace_path = scaffold_workspace(workspace_name)
+    new_project_options = DgWorkspaceNewProjectOptions.get_raw_from_cli(
+        use_editable_dagster,
+        use_editable_components_package_only,
+    )
+
+    workspace_path = scaffold_workspace(workspace_name, new_project_options)
```

Comment:
little weird to use project options when scaffolding something that is not a project, is it just "scaffold_options" maybe?

Reply:
I can see how it's a little confusing but these options don't really modulate the scaffolding of the workspace, they are just a setting that gets written in the workspace config. Maybe wrap it in an additional layer and call it "workspace config"?